src/components/orderItem.js

```diff
@@ -18,7 +41,7 @@ class OrderItem extends React.Component {
                 <div className="uk-width-1-2">
                     <div className="uk-grid">
                         <div className="uk-width-1-3">
-                            <img alt={this.props.item.image} src={this.props.item.image} width="30em" height="30em" />
+                            <img src={this.props.item.image} width="30em" height="30em" />
```

Comment:
Good catch on the alt tag.
Might want to add an alt tag using the product name, for example:

```jsx
<img alt={"Photo of " + this.props.item.name} src={this.props.item.image} ... />
```